content/python/concepts/modules/modules.md

```diff
@@ -56,3 +56,26 @@ from bs4 import BeautifulSoup as bs
 Aliasing is especially convenient for shortening module names and managing the local namespace where our code executes.
 
 Once a module is imported, it is within the scope of the program and it can be accessed in the local namespace.
+
+## Python Standard Modules
+
+Python comes with a number of different built-in modules that provide a variety of functions. They include:
+
+- The [`collections`](https://www.codecademy.com/resources/docs/python/collections-module) module provides additional collection types.
+- The [`functools`](https://www.codecademy.com/resources/docs/python/functools-module) module provides functions supporting a [functional programming](https://www.codecademy.com/resources/docs/general/functional-programming) approach.
+- The [`glob`](https://www.codecademy.com/resources/docs/python/glob-module) module allows matching file paths per [Unix](https://www.codecademy.com/resources/docs/general/unix) shell rules.
+- The [`json`](https://www.codecademy.com/resources/docs/python/json-module) module provides functions for dealing with [JSON](https://www.codecademy.com/resources/docs/general/json) objects.
+- The [`math`](https://www.codecademy.com/resources/docs/python/math-module) module provides useful mathematical functions.
+- The [`random`](https://www.codecademy.com/resources/docs/python/random-module) module provides functions for dealing with random numbers.
+- The [`time`](https://www.codecademy.com/resources/docs/python/time-module) module provides various functions for dealing with time.
+
+## Python Third-Party Modules
+
+Python has a very broad selection of third-party modules that are devoted to particular tasks.
+
+These are third-party Python modules that have their own topic entries:
+
+- [NumPy](https://www.codecademy.com/resources/docs/numpy): a popular open source Python library used for complex mathematical operations and multi-dimensional [arrays](https://www.codecademy.com/resources/docs/general/array).
+- [Pandas](https://www.codecademy.com/resources/docs/pandas): a popular open source Python module used for data analysis and manipulation.
```

Comment:
```suggestion
- [NumPy](https://www.codecademy.com/resources/docs/numpy): a popular open-source Python library used for complex mathematical operations and multi-dimensional [arrays](https://www.codecademy.com/resources/docs/general/array).
- [Pandas](https://www.codecademy.com/resources/docs/pandas): a popular open-source Python module used for data analysis and manipulation.
```

Reply:
Docs entries use "open-source" more so than "open source"

Reply:
I agree with this suggestion. And there actually is a place for both variations! If used as an adjective preceding a noun (as is the case, here), it should be "open-source". But if used more like a concept or noun (e.g., "Open source is...", "this is open source"), the hyphen should not be included.